### ISSUE TITLE
/ui: Line height setting in A11y menu should use unitless numbers

### ISSUE THREAD

**Issue by saabi** [nestauk/svizzle#484]
The A11y menu currently uses `rem` units internally for specifying line height adjustments. This results in line heights being relative to the root font size when it should be proportional to an element's local font size.
